account_test.go

```diff
@@ -26,6 +27,131 @@ import (
 	"time"
 )
 
+// memoryStorage is an in-memory storage implementation with known contents *and* fixed iteration order for List.
+type memoryStorage struct {
+	contents []memoryStorageItem
+}
```

Comment:
FWIW: Apart from needing this for controlling the iteration order of `List`, this might also be faster in general than using an actual file system for unit tests. And, there's good references that say that I/O in unit tests should be banned. :)

Reply:
Approved :100: